base/applications/regedit/framewnd.c

```diff
@@ -1474,17 +1463,11 @@ static BOOL _CmdWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
     return result;
 }
 
-/********************************************************************************
- *
- *  FUNCTION: FrameWndProc(HWND, unsigned, WORD, LONG)
- *
- *  PURPOSE:  Processes messages for the main frame window.
- *
- *  WM_COMMAND  - process the application menu
- *  WM_DESTROY  - post a quit message and return
+/* PURPOSE: Processes messages for the main frame window
```

Comment:
```suggestion
/*
 * PURPOSE: Processes messages for the main frame window
```

Reply:
feels like a wasted line to me, but if it will make an approval appear, then fine. Committed.